deployment/terraform/assets/provisioners/app.sh

```diff
@@ -17,6 +19,9 @@ do
       sudo sh -c 'echo "deb [signed-by=/usr/share/keyrings/postgres-archive-keyring.gpg] http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list' && \
       sudo apt-get -y update && \
       sudo apt-get install -y mysql-client-8.0 && \
+      sudo apt-get remove -y needrestart && \
+      sudo apt-get install -y -o Dpkg::Options::="--force-confold" binutils git cargo libssl-dev make pkg-config && \
+      git clone https://github.com/aws/efs-utils && cd efs-utils && make deb && sudo sudo apt-get install -y -o Dpkg::Options::="--force-confold" ./build/amazon-efs-utils-*.deb && cd - && \
```

Comment:
Can we just host the debian file somewhere that we can download and install? This is adding more complications to an already complicated and error-prone provisioning step.

Reply:
Yep, that works.

Reply:
Agree that we should try to make this a bit more reliable. At least using a specific tag of the repo. But I'm on board with the fork idea.

Reply:
Let's make this change as well to avoid making the debian. Just download from somewhere and do a `dpkg -i`